core/services/mongodb-remote/src/main/java/com/mongodb/stitch/core/services/mongodb/remote/sync/internal/DataSynchronizer.java

```diff
@@ -1543,20 +1548,21 @@ public void desyncDocumentFromRemote(
       final BsonValue documentId
   ) {
     syncConfig.removeSynchronizedDocument(namespace, documentId);
+    getLocalCollection(namespace).deleteOne(getDocumentIdFilter(documentId));
     triggerListeningToNamespace(namespace);
   }
 
   /**
    * A document that is paused no longer has remote updates applied to it.
    * Any local updates to this document cause it to be resumed. An example of pausing a document
    * is when a conflict is being resolved for that document and the handler throws an exception.
-   *
+   * <p>
```

Comment:
Stray?

Reply:
See above. Though technically, this is also fine.

Reply:
👍